pyjobshop/solvers/ortools/Variables.py

```diff
@@ -474,6 +536,65 @@ def _make_sequence_variables(self) -> dict[ResourceIdx, SequenceVar]:
 
         return variables
 
+    def _make_overlap_variables(self) -> list[list[OverlapVar]]:
+        """
+        Creates the overlap variables.
+        """
+        model, data = self._model, self._data
+        variables = []
+
+        for mode_idx, mode in enumerate(data.modes):
+            breaks = [
+                brk
+                for res_idx in mode.resources
+                for brk in getattr(data.resources[res_idx], "breaks", [])
+            ]
+            breaks = merge(breaks)  # super breaks
+            task_var = self._task_vars[mode.task]
+            mode_var = self._mode_vars[mode_idx]
+            overlap_vars = []
+
+            for start, end in breaks:
+                before = model.new_bool_var("")
+                model.add(task_var.end <= start).only_enforce_if(
+                    [before, mode_var]
```

Comment:
If we don't include a mode variable here, it constrains the start/end variables of all assignment variables.